cache/cache_test.cc

```diff
@@ -955,6 +956,90 @@ TEST_P(CacheTest, GetChargeAndDeleter) {
   cache_->Release(h1);
 }
 
+bool AreTwoCacheKeysOrdered(Cache* cache) {
+  std::vector<std::string> keys;
+  const auto callback = [&](const Slice& key, Cache::ObjectPtr /*value*/,
+                            size_t /*charge*/,
+                            const Cache::CacheItemHelper* /*helper*/) {
+    keys.push_back(key.ToString());
+  };
+  cache->ApplyToAllEntries(callback, /*opts*/ {});
+  EXPECT_EQ(keys.size(), 2U);
+  EXPECT_NE(keys[0], keys[1]);
+  return keys[0] < keys[1];
+}
+
+TEST_P(CacheTest, CacheUniqueSeeds) {
+  // kQuasiRandomHashSeed should generate unique seeds (up to 2 billion before
+  // repeating)
+  UnorderedSet<uint32_t> seeds_seen;
+  seeds_seen.reserve(20000);
```

Comment:
Curious how did you choose 20000 instead of a smaller number to ensure we see both orders in the UT

Reply:
If it were fully random (possible repeats), we need on the order of sqrt(number of possible values) to have a good chance of observing a collision. So ~2^14 is not far from the sqrt(2^31).

Reply:
~50ms of CPU is sufficient for this "stress test in a unit test" to do its job, with not high but decent sensitivity. (My personal rule of thumb is < 1s for a "stress test in a unit test," preferably much less.)